File: SeongYeon/js/programmers/bestSet.js
function solution(n, s) {
  if (n > s) return [-1];

  const devide = Math.floor(s / n);
  const answer = new Array(n).fill(devide);
  let rest = s % n;

  for (let i = n - rest; i < n; i++) answer[i] += 1;

  return answer;
}

File: JiHwan/programmers/programmers_highscore_kit/best_set.py
def solution(n, s):
    answer = [0, 0]

    if n > s:
        return [-1]
    else:
        if s % n == 0:
            vector = s // n
            answer[0] = vector
            answer[1] = vector
        else:
            num = s // n

            answer[0] = num
            answer[1] = num + 1

    return answer